src/components/Follow.test.tsx

```diff
@@ -10,17 +11,21 @@ import {
   BOB_PUBLIC_KEY,
   setup,
   follow,
+  bobsNip05Identifier,
+  finalizeEventWithoutWasm,
 } from "../utils.test";
-import { KIND_CONTACTLIST } from "../nostr";
+import { KIND_CONTACTLIST, newTimestamp } from "../nostr";
 import { Follow } from "./Follow";
+// import { useNip05Query } from "./useNip05Query";
+
+afterEach(() => {
+  jest.useRealTimers();
```

Comment:
what's this?

Reply:
i set a timeout im Follow so that the error message doesn't appear instantly (but 500ms after the query doesn't find any nip05events)

Reply:
nip11 also has a timeout (searchDebounce) which is set in the Provider

Reply:
set timeout in provider then this is not necessary